recipe/run_test.bat

```diff
@@ -4,5 +4,5 @@ ncgen -o in.nc in.cdl || exit 1
 
 ncks -H --trd -v one in.nc || exit 1
 
-ncap2 -O -v -s 'erf_one=float(gsl_sf_erf(1.0f));print(erf_one,"%g")' in.nc foo.nc || exit 1
-
+ncap2 -O -v -s "erf_one=float(gsl_sf_erf(1.0f));" in.nc foo.nc || exit 1
+ncks -v erf_one foo.nc || exit 1
```

Comment:
@pedro-vicente and @czender replace the single quotes with double and removing the print, tried many escape option and quote combinations but I could not make it work, we get the expected results. See the `erf_one` value from the following `ncks` call.

TL;DR `ncap2` works as expected but users most be warned that examples from the manual may need some tweaking to make them work in the Windows CLI.